README.org

```diff
@@ -45,6 +51,17 @@ rights to the work you contribute.
 - [[WebKit2GTK+ Stable API reference]]
 - [[http://common-lisp.net/project/cffi/manual/html_node/index.html][CFFI User Manual]]
 
+** Change Log
+*** 3.0.0
+- Make =webkit-web-view-evaluate-javascript= process the returned values
+  and turn them into the matching Lisp values (see =jsc-value-to-lisp=
+  docstring for transformation rules).
+- The callback to the =webkit-web-view-evaluate-javascript= now accepts
+  two required arguments -- the Lispy data, transformed from the
+  results of code evaluation, and an original [[https://webkitgtk.org/reference/jsc-glib/2.32.1/JSCValue.html][JSCValue]] it was
+  transformed from.
```

Comment:
This got me to think, what if the callback took `(jsc-value & optional lisp-value)` instead?  Wouldn't be backward compatible?

Reply:
Won't work. Both `(funcall (lambda (a) a) 1 2)` and `(apply (lambda (a) a) 1 2 nil)` signal errors. We use strict one-argument callbacks like `(lambda (a) a)` everywhere, thus this code will become erroneous even with optional arguments. Thus, it's inevitably backwards incompatible.